src/mlpack/core/optimizers/sgd/sgd.hpp

```diff
@@ -16,6 +16,7 @@
 #include <mlpack/prereqs.hpp>
 #include <mlpack/core/optimizers/sgd/update_policies/vanilla_update.hpp>
 #include <mlpack/core/optimizers/sgd/update_policies/momentum_update.hpp>
+#include <mlpack/core/optimizers/sgd/update_policies/smorms3_update.hpp>
```

Comment:
I don't think we have to include the smorms3_update header file here, we include vanilla and momentum header files, to create an alias where this to classes are necessary.